R/compute_clusters.R

```diff
@@ -1,4 +1,23 @@
 
+#' @description
+#' Sort tibble based on sample_names
+#' @export
+sort_data <- function(sample_names, feature_tables){
+  index <- c()
+  for(i in seq_along(sample_names))
+  {
+    for(x in seq_along(sample_names))
+    {
+      if(sample_names[i] == feature_tables[[x]]$sample_id[1])
+      {
+        index <-  append(index, x)
+      }
+    }
+  }
+  feature_tables <- feature_tables[index]
+  return(feature_tables)
+}
+
```

Comment:
I think this could be done in a more elegant way than using nested `for` loop. I would do it this way:
1. create vector of sample_ids taken from feature_tables
2. compare the vectors (I think [match](https://rdrr.io/r/base/match.html) is what you need), creating the index needed for the reordering
3. reorder the feature_tables

Reply:
done in [9fadfd6](https://github.com/RECETOX/recetox-aplcms/pull/177/commits/9fadfd67574f763aab091301a974d6ee05259d50)